pkg/auth/sessions/combined_sessions.go

```diff
@@ -166,9 +166,23 @@ func (cs *CombinedSessionStore) DeleteSession(w http.ResponseWriter, r *http.Req
 		}
 	}
 
+	cookieSession := cs.getCookieSession(r)
+	if sessionToken, ok := cookieSession.sessionToken.Values["session-token"]; ok {
+		cs.serverStore.DeleteBySessionToken(sessionToken.(string))
+	}
+
+	if refreshToken, ok := cookieSession.refreshToken.Values["refresh-token"]; ok {
+		cs.serverStore.DeleteByRefreshToken(refreshToken.(string))
+	}
```

Comment:
so previously we have been waiting for the token to expire ?

Reply:
Yes. This is still not great since, as I've noticed, the session deletion code will likely never be called.
A fix should come in a different PR.